modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItDataTypesTest.java

```diff
@@ -315,8 +315,6 @@ public void testDecimalLiteral() {
         assertQuery("SELECT CASE WHEN false THEN DECIMAL '1.00' ELSE DECIMAL '0.0' END")
                 .returns(new BigDecimal("0.0")).check();
 
-        assertQuery("SELECT DECIMAL \"10\" FROM (SELECT 1 as decimal) tmp").returns(1).check();
```

Comment:
Why it has been deleted?

Reply:
this particular case verifies that `DECIMAL \"10\"` is parsed as column name and alias and not like decimal literal. In this patch keyword `DECIMAL` becomes reserved, rendering such construction not possible anymore